### ISSUE TITLE
Pre-train accuracy

### ISSUE THREAD

**Issue by Vickeyhw** [megvii-model/DetNAS#28]
How does your accuracy on the training set change during the process of pre-training on ImageNet?  I found that my training accuracy improves very slowly when training a supernet defined by myself, Is it normal?

**Comment by yukang2017** [comment]
Sorry for the late reply. Wish that you have already solved this issue. If not, would you please provide more information on how you define the supernet and the training accuracy. Actually, the supernet accuracy requires not so high.

**Comment by yukang2017** [comment]
Feel free to reopen this issue if you have any questions.
